client/my-sites/checkout/composite-checkout/components/titan-terms-of-service.jsx

```diff
@@ -0,0 +1,60 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { hasTitanMail } from 'calypso/lib/cart-values/cart-items';
+import Gridicon from 'calypso/components/gridicon';
+
+/* eslint-disable wpcalypso/jsx-classname-namespace */
+
+class TitanTermsOfService extends React.Component {
+	render() {
+		const { cart, translate } = this.props;
```

Comment:
Super minor: I have no objections to using a React class component, but since this component is just a `render` function, it might make its props slightly more clear by making this a functional component instead. You'd just need to change these top lines to the following and remove some parenthesis:

```js
function TitanTermsOfService( { cart, translate } ) {
```

Reply:
Thanks for the suggestion! I am more than happy to simplify here. ✅